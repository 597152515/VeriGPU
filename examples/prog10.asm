addi x31, x0, 1000

li x1, 123
li x2, 100
sw x1, 0(x2)

li x1, 222
li x2, 110
sw x1, 0(x2)

li x2, 100
lw x3, 0(x2)
sw x3, 0(x31)

li x2, 110
lw x3, 0(x2)
sw x3, 0(x31)

halt
