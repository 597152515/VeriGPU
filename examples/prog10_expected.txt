out 0 00000000000000000000000001111011 0000007b 123
out 1 00000000000000000000000011011110 000000de 222